apps/emqx_modules/test/emqx_telemetry_SUITE.erl

```diff
@@ -66,17 +66,35 @@ init_per_testcase(t_get_telemetry, Config) ->
         end
     ),
     Config;
+init_per_testcase(t_advanced_mqtt_features, Config) ->
+    OldValues = application:get_env(emqx_telemetry, advanced_mqtt_features_in_use, #{}),
+    application:set_env(
+        emqx_telemetry,
+        advanced_mqtt_features_in_use,
+        #{
+            delayed => false,
+            topic_rewrite => false,
+            retained => false,
+            auto_subscribe => false
+        }
+    ),
+    [{old_values, OldValues} | Config];
 init_per_testcase(_Testcase, Config) ->
     TestPID = self(),
     ok = meck:new(httpc, [non_strict, passthrough, no_history, no_link]),
-    ok = meck:expect(httpc, request, fun(Method, URL, Headers, Body) ->
+    ok = meck:expect(httpc, request, fun(
+        Method, {URL, Headers, _ContentType, Body}, _HTTPOpts, _Opts
+    ) ->
         TestPID ! {request, Method, URL, Headers, Body}
     end),
```

Comment:
Is it automatic formatting?

Reply:
Yes, I ran `scripts/format_app.py -a apps/emqx_modules -f` and it gave me this.

Reply:
:see_no_evil: